test/Tester/Facet/StorageFeatureAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Orleans;
+using Orleans.Runtime;
+
+namespace Tester
+{
+    [AttributeUsage(AttributeTargets.Property)]
```

Comment:
Agreed, but we'd need some additional wiring.  Best to keep them constrained to the only supported pattern for now.  Do you disagree?